client/components/DisplayTest/index.jsx

```diff
@@ -160,36 +178,74 @@ class DisplayTest extends Component {
         }
 
         return (
-            <Modal
-                show={this.state.showConfirmModal}
-                onHide={this.handleModalCloseClick}
-                centered
-                animation={false}
-            >
-                <Modal.Header closeButton>
-                    <Modal.Title>{modalTitle}</Modal.Title>
-                </Modal.Header>
-                <Modal.Body>{action}</Modal.Body>
-                <Modal.Footer>
-                    <Button
-                        variant="secondary"
-                        onClick={this.handleModalCloseClick}
-                    >
-                        Cancel
-                    </Button>
-                    <Button
-                        variant="secondary"
-                        onClick={this.handleModalConfirmClick}
-                    >
-                        Continue
-                    </Button>
-                </Modal.Footer>
-            </Modal>
+            <>
```

Comment:
omg, what is this? replacement for fragment?

Reply:
Yes